.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,10 @@
 Thank you for your interest in contributing! For general guidelines, please refer to
 the [contributing guide](https://github.com/GoogleContainerTools/jib/blob/master/CONTRIBUTING.md).
 
-Before filing a pull request, make sure to do the following:
-
-- [ ] Create a new issue at https://github.com/GoogleContainerTools/jib/issues/new/choose.
```

Comment:
I think these checkboxes are still very valid, can we keep them? I know people are not really following them right now but it still helps.

Reply:
Yup! I've included a combination of what we had before (with verification of integration tests and checkstyle) and some of the new content.